CHANGELOG

```diff
@@ -1,3 +1,7 @@
+snowplow-unified 0.4.4 (2024-06-25)
+---------------------------------------
```

Comment:
Why is it needed here? Thought we can test this upon opening a new pr against main AFTER this is live?

Reply:
Accident, shouldn't be here :D